test/unit/model/scope.test.ts

```diff
@@ -154,7 +154,7 @@ describe('src/model/scope', (): void => {
       expect(scopes.length).toEqual(2)
 
       // Action
-      await scopeDB.delete(tempScope.consentId)
+      await scopeDB.deleteAll(tempScope.consentId)
 
```

Comment:
after `deleteAll` you should check that in the database there are no scopes for `tempScope.consentId`